core/conf/web.conf

```diff
@@ -8,8 +8,15 @@
 		# google credential path for the service account, relative to this `core/conf/`.
 		"credentialPath": ""
 	},
-	"python" :{
+	"python": {
 	    # python3 executable path
 	    "path": ""
+
+	    "log": {
+	    	# python3 log diretory
+	        "dir": "/tmp/",
+	        # python3 log level
+	        "level": "ERROR"
```

Comment:
lets change this to INFO or whatever is the least. by default lets log everything

Reply:
changed.